docs/Creating_binary_packages.rst

```diff
@@ -0,0 +1,20 @@
+.. _creating_binary_pages:
+
+Creating Binary Packages
+========================
+
+As an experimental feature, support for `FPM <https://github.com/jordansissel/fpm>`_ has been integrated into EasyBuild. This creates a dependency on Ruby and a few other RubyGems and they have been provided through `Ruby-2.1.5.eb <https://github.com/rjeschmi/easybuild-easyconfigs/blob/a297788484eb778faeef5c9f335a58cc3b33c9b4/easybuild/easyconfigs/r/Ruby/Ruby-2.1.5.eb>`_.
```

Comment:
Maybe I'll outline the mainline documentation for experimental under Advanced and reference that?